docs/source/commands/subgraphs.mdx

```diff
@@ -144,6 +144,10 @@ View full details at https://studio.apollographql.com/graph/my-supergraph/servic
 
 ### `subgraph publish`
 
+<Note>
+See also the guide for [Using Subgraph Keys in a Deployment Pipeline](/graphos/resources/guides/getting-started-with-pipelines-and-keys).
+</Note>
+
```

Comment:
This feels like an unnecessary detour. Can we incorporate the need for a subgraph key somewhere in the content itself when it's relevant?

Reply:
The `AuthNotice` below shows: 

> This command requires [authenticating Rover with GraphOS](https://www.apollographql.com/docs/rover/configuring/#authenticating-with-graphos)

The page it links out to already talks about when to use which API key so I think we should be okay to remove this detour!